### ISSUE TITLE
Request: Select multiple markers

### ISSUE THREAD

**Issue by btidoug** [jmelosegui/GooglemapMvc#49]
Is there any way possible to add the ability of "clicking and dragging" a selection box around a group of markers and "select them"?

There are 4 markers in the image - is there anyway to select them all by dragging a selection box or polygon, etc.?

LOVE THE PLUGIN SO FAR!

![select](https://cloud.githubusercontent.com/assets/6748163/8723076/e6bc5b88-2b98-11e5-9d29-f522368c62f8.png)

**Comment by jmelosegui** [comment]
I don't think you can do that with google map api. So please if you find a sample of that feature let me know and I will try to implement it inside my control. In the meanwhile I will close this issue.

Feel free to reopen it if you find that sample on [google maps api developer reference](https://developers.google.com/maps/documentation/javascript/3.exp/reference)

**Comment by btidoug** [comment]
Not a sample at google api but something that could be done with polygons maybe?

http://www.daftlogic.com/sandbox-google-maps-marker-lasso-search-tool.htm

**Comment by btidoug** [comment]
Here's another sample of polygon.

http://www.mvjantzen.com/blog/?p=3190

**Comment by btidoug** [comment]
1 more ...

http://stackoverflow.com/questions/17673580/google-maps-drag-rectangle-to-select-markers

**Comment by btidoug** [comment]
I think I could get it to work if the "map" had events for Shift and Ctrl ... not the marker events - the map events.

**Comment by btidoug** [comment]
Might also need a "mouse up" even on the map - not the marker.

**Comment by btidoug** [comment]
Would you be open to "donations" to finish this enhancement?

**Comment by jmelosegui** [comment]
Right now I have no time for this project. But I am open to pull requests.

**Comment by btidoug** [comment]
$50?  $100?

**Comment by btidoug** [comment]
Any time to work on this yet?

**Comment by jmelosegui** [comment]
my next thing here is to make it compatible with asp.net core

**Comment by btidoug** [comment]
I would use the feature in asp.net core too

**Comment by maxitimes** [comment]
Awesome, can't wait till .Net Core version is ready.

Thank you

**Comment by btidoug** [comment]
Any luck on bringing this feature to life?  If not, do you have any recommendations on how to implement this "outside" your solution?  My app really could benefit from this enhancement!  I'll donate money!
